jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/BlobCheckerIntegrationTest.java

```diff
@@ -28,11 +29,12 @@
 public class BlobCheckerIntegrationTest {
 
   @ClassRule public static LocalRegistry localRegistry = new LocalRegistry(5000);
+  private final EmptyBuildLogger buildLogger = new EmptyBuildLogger();
```

Comment:
Let's be consistent. `static final ... BUILD_LOGGER`?

Reply:
Or even just a singleton: `EmptyBuilderLogger.INSTANCE`.